### PR TITLE
38 tests for one shot device

### PR DESCRIPTION
**Pull request by jacekroszkowiakdev**
Tests for one-shot-device completed

### CODE REVIEW

File: one-shot-device/src/devices/device-registration.controller.spec.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import { HttpModule } from '@nestjs/axios';
+import { MongooseModule, getModelToken } from '@nestjs/mongoose';
+import { DeviceRegistration, DeviceRegistrationSchema, DeviceRegistrationDocument } from './schemas/device-registration.schema';
+import { Test, TestingModule } from '@nestjs/testing';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { channelMock, identityMock, mockDeviceRegistration, nonceMock } from './mocks';
+import { ChannelClient, IdentityClient } from 'iota-is-sdk';
+import { Model } from 'mongoose';
+
+jest.setTimeout(40000);
```
Comment (dominic22) [inline one-shot-device/src/devices/device-registration.controller.spec.ts:13]: please remove

File: one-shot-device/src/devices/device-registration.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+import { MongooseModule, getModelToken } from '@nestjs/mongoose';
+import { DeviceRegistration, DeviceRegistrationDocument, DeviceRegistrationSchema } from './schemas/device-registration.schema';
+import { Model } from 'mongoose';
+import { Test, TestingModule } from '@nestjs/testing';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { badNonceMock, identityMock, channelMock } from './mocks';
+import { CreateChannelResponse } from '../../node_modules/iota-is-sdk/lib/models/types/request-response-bodies';
+import { IdentityJson } from '../../node_modules/iota-is-sdk/lib/models/types/identity';
+
+jest.setTimeout(40000);
```
Comment (dominic22) [inline one-shot-device/src/devices/device-registration.service.spec.ts:14]: please remove

**Review by dominic22 — APPROVED** [review]

File: one-shot-device/src/devices/device-registration.service.spec.ts (hunk 2 above)
Comment (jacekroszkowiakdev) [reply one-shot-device/src/devices/device-registration.service.spec.ts:14]: Resolved

File: one-shot-device/src/devices/device-registration.controller.spec.ts (hunk 1 above)
Comment (jacekroszkowiakdev) [reply one-shot-device/src/devices/device-registration.controller.spec.ts:13]: Resolved

**Pull request merged**